apache-jsp/pom.xml

```diff
@@ -10,7 +10,7 @@
   <url>http://www.eclipse.org/jetty</url>
   <packaging>jar</packaging>
   <properties>
-    <bundle-symbolic-name>${project.groupId}.${project.artifactId}</bundle-symbolic-name>
+    <bundle-symbolic-name>${project.groupId}.apache.jsp</bundle-symbolic-name>
```

Comment:
Why do you wish to change the bundle symbolic name? This is significant in osgi, so unless there's an urgent reason to change it, I'd rather leave it as is.

Reply:
OK but as discussed we need to note this somewhere as bundle names are significant in osgi and users may will have other code that relies on the old bundle name.